source/documentation/breaking-changes.html.md.erb

```diff
@@ -40,3 +40,75 @@ These breaking changes are coming soon or have recently been released:
 
 * [The syntax for CSS custom property values changed](breaking-changes/css-vars)
   in Dart Sass 1.0.0, LibSass 3.5.0, and Ruby Sass 3.5.0.
+
+## Early Opt-In
```

Comment:
It's probably better to document this on https://sass-lang.com/documentation/cli/dart-sass and link to it from here.

Reply:
Done